zkvm/docs/zkvm-spec.md

```diff
@@ -1598,7 +1599,17 @@ All unassigned instruction codes are interpreted as no-ops. This are reserved fo
 
 See [Versioning](#versioning).
 
+## Transaction Encoding
 
+A [Transaction](#transaction) is serialized as follows:
+
+```
+        SerializedTx = TxHeader || LE32(len(Program)) || Program || Signature || Proof
+        TxHeader = <24 bytes>
+        Program = <len(Program) bytes>
+        Signature = <64 bytes>
+        Proof = <14 * 32 + len(InnerProductProof) bytes>
```

Comment:
I try to use `·` for multiplication everywhere in the docs for the sake of typography.
```suggestion
        Proof = <14·32 + len(InnerProductProof) bytes>
```